core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -388,6 +388,16 @@ public void setStateTest() throws Exception {
   }
 
   /**
+   * Tests the permission bits are 0755 for directories and 0644 for files by default.
+   */
+  @Test
+  public void permissionTest() throws Exception {
+    mFileSystemMaster.createFile(NESTED_FILE_URI, sNestedFileOptions);
+    Assert.assertEquals(0755, mFileSystemMaster.getFileInfo(NESTED_URI).getPermission());
```

Comment:
the reason we didn't put 0755 a constant here is because it is calculated from applying umask for dir on 0777 (this is in a constant). same for 0644.  In other words, they are calculated results (deep in the call stack), rather than being used directly. So I don't really see a clean way to define 0755 or 0644 themselves a constant, maybe only in tests?